webapp/app/logging.py

```diff
@@ -11,12 +11,16 @@ def configure_logging():
         dictConfig(json.load(f))
 
 
-class AddRequestIdFilter(logging.Filter):
-    """Add the request ID to the log (if present)."""
+class AddRequestInformationFilter(logging.Filter):
+    """Add the important request information (id, url) to the log (if present)."""
```

Comment:
I wouldn't list which bits are logged (id, url) in the comment, it's already out of date! :P

Reply:
🙈